2009-08-08  Yavor Doganov  <yavor@gnu.org>  (tiny change)

	* Documentation/library-combo.7: Add whatis entry.


--- gnustep-make-2.2.0.orig/Documentation/library-combo.7
+++ gnustep-make-2.2.0/Documentation/library-combo.7
@@ -2,6 +2,8 @@
 .\" groff -man -Tascii library-combo.7
 .\" 
 .TH library-combo 7 "20/12/2007" gnustep-make "GNUstep System Manual"
+.SH NAME
+library-combo \- GNUstep Make support for cross-compilation.
 .SH DESCRIPTION
 This manual page is about cross-compilation and fat binaries. Fat binaries are packages that you can supply which contain more the one binary of your program (e.g. your application). So you can support multiple platforms with a single package.
 .PP
